Inav: update page (#10026)
tldr-pages/tldr: pages/linux/lnav.md

- Validate the format of log files against the configuration and report any errors:

`lnav -C {{path/to/log_directory}}`

> - Load all of the files in '/var/log':

Reviewer: ```suggestion
-  View logs of  the directory '/var/log':
```
I have a minor suggestion
> I don't think this example is necessary, but it is fine IG.

> - View logs of the directory '/var/log':

Reviewer 2: I think the first example includes the usage of this. There is no need to add this new example.

Author: I'll remove it

Reviewer 2: Yes, I mean this example(line 18~20) is a specific usage of the example one. So we do not accept this  new update.
